doc/installation.rst

```diff
@@ -33,6 +33,7 @@ SmartRedis is tested with the following compilers on a daily basis:
     * - Compilers (tested daily)
     * - GNU (GCC/GFortran)
     * - Intel oneAPI (icx/icpx/ifort)
+    * - NVHPC
     * - Apple Clang
```

Comment:
Might need to move Clang to the less frequent section now

Reply:
I don't want to remove this yet, since it still does compile. Plus, I think because the container isn't used anymore, we might be able to restore the Apple Clang compiler chain.